uni-bern-institut-fur-musikwissenschaft-note.csl

```diff
@@ -0,0 +1,543 @@
+  </info>
+  <locale xml:lang="de">
+    <terms>
+      <term name="editor" form="verb-short">hrsg. v.</term>
+      <term name="anonymous" form="short">o.A.</term>
+      <term name="number-of-volumes" form="short">Bde.</term>
+      <term name="sub verbo" form="short">Art.</term>
+      <term name="issue" form="short">H.</term>
+      <term name="and">u.</term>
+    </terms>
+    <date form="numeric">
+      <date-part name="day" suffix="."/>
+      <date-part name="month" form="numeric" suffix="."/>
+      <date-part name="year"/>
+    </date>
+  </locale>
+  <macro name="sort">
+    <choose>
+      <if match="none" variable="author composer editor">
+        <text macro="anon"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="sort-2">
+    <choose>
+      <if type="motion_picture" match="none">
+        <names variable="composer">
+          <name delimiter=" / " name-as-sort-order="all"/>
+          <substitute>
+            <names variable="author">
+              <name delimiter=" / " name-as-sort-order="all"/>
+              <label prefix=" (" suffix=")" form="short"/>
+              <substitute>
+                <names variable="editor">
+                  <name delimiter=" / " name-as-sort-order="all"/>
+                  <label prefix=" (" suffix=")" form="short"/>
+                </names>
+              </substitute>
+            </names>
+            <text macro="anon"/>
+          </substitute>
+        </names>
+      </if>
+      <else>
+        <text variable="title"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="cite-short">
+    <group delimiter=", ">
+      <text macro="primary-contributor-short"/>
+      <text macro="title-short"/>
+      <text macro="locator"/>
+    </group>
+  </macro>
+  <macro name="cite-ibid-with-locator">
+    <group delimiter=", ">
+      <text macro="cite-ibid"/>
+      <text macro="locator"/>
+    </group>
+  </macro>
+  <macro name="cite-ibid">
+    <text term="ibid" form="short"/>
+  </macro>
+  <macro name="author_title-citations">
+    <group delimiter=", ">
+      <text macro="primary-contributor-citations"/>
+      <text macro="title_volumes"/>
+    </group>
+  </macro>
+  <macro name="author_title-bibliography">
+    <group delimiter=": ">
+      <text macro="primary-contributor"/>
+      <text macro="title_volumes"/>
+    </group>
+  </macro>
+  <macro name="reference-publication-details">
+    <text macro="details-motion-picture" prefix=", "/>
+    <text macro="genre-format" prefix=", "/>
+    <text macro="container" prefix=", "/>
+    <text macro="other-contributors" strip-periods="false" prefix=", "/>
+    <text macro="series_number" prefix=" "/>
+    <text macro="publisher_location_year" prefix=", "/>
+    <text macro="volume" prefix=", "/>
+    <text macro="locators" prefix=", "/>
+    <text macro="archival-details" prefix=", "/>
+  </macro>
+  <macro name="primary-contributor">
+    <choose>
+      <if type="motion_picture" match="none"/>
```

Comment:
This does nothing at all. I assume you want to move the 
```
</if>
</choose>
``` 
down?

Reply:
Hmm; I don't really know what this should do. Have removed it completely for now. Probably a left-over from an earlier version and not need anymore...